ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/dot1x.md

```diff
@@ -33,4 +52,43 @@
       radius_av_pair:
         service_type: <bool>
         framed_mtu: <int; 68-9236>
+
+      # Configure AAA parameters.
+      aaa:
+
+        # Configure AAA timeout options.
+        unresponsive:
+          eap_response: <str; "success" | "disabled">
+
+          # Set action for supplicant when AAA times out.
+          action:
+
+            # Use results from a previous AAA response.
+            apply_cached_results: <bool>
+            cached_results_timeout:
+              time_duration: <int; >=1>
+              time_duration_unit: <str; "days" | "hours" | "minutes" | "seconds"; required>
```

Comment:
i know we cant add the limit in values but can we add it in comment

Reply:
Fixed